src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/RunReportPageTests.kt

```diff
@@ -120,4 +127,60 @@ class RunReportPageTests : IntegrationTest()
                 ReportWithDate("minimal", repo.getLatestVersion("minimal").date)
         )
     }
+
+    @Test
+    fun `can get running reports details`()
+    {
+        val instant = Instant.now()
+        val fakeReportRunLog = ReportRunLog(
+                "test@example.com",
+                instant,
+                "q123",
+                mapOf("annex" to "production", "source" to "production"),
+                mapOf("name" to "cologne", "value" to "memo"),
+                "branch",
+                "commit",
+                "complete",
+                "logs",
+                "1233")
+
+        val mockContext: ActionContext = mock {
+            on { params(":key") } doReturn "fakeKey"
+        }
+
+        val mockRepo = mock<ReportRunRepository> {
+            on { getReportRun("fakeKey") } doReturn fakeReportRunLog
+        }
+
+        val sut = ReportRunController(mockContext, mockRepo)
+        val result = sut.getRunningReportLogs()
+        assertThat(result).isEqualTo(fakeReportRunLog)
+    }
```

Comment:
This looks like a controller test - not an integration test - as all the controller's parameters are mocked. You can move it to `RunReportTests.kt` if it's testing anything that isn't there already.